config/optional/responsive_image.styles.az_full_width_background.yml

```diff
@@ -0,0 +1,30 @@
+langcode: en
```

Comment:
Dumb question: Does this responsive image style need to be optional config?

Reply:
Maybe it should be moved into the `az_media` module's config directory?

Reply:
I don't think it needs to be optional since the responsive image module is now required.

Reply:
I'll move it to the module config dir.